Makefile

```diff
@@ -251,9 +248,6 @@ preconfigure-mainnet:
 	# Configure the OFTs
 	$(CONFIGURE_OFT) $(CONFIGURE_ARGS_COMMON) --oapp-config $(CONFIG_BASE_PATH)/oft-token.config.ts --signer deployer
 
-	# Set the admin to secondary role(MULTISIG) so our calls as owner get through
-	$(CONFIGURE_USDC_SET_ADMIN) $(CONFIGURE_ARGS_COMMON) --oapp-config $(CONFIG_BASE_PATH)/usdc-admin.config.ts --signer usdcAdmin
-
```

Comment:
Since we are deploying externally, the admin will already be set to the multisig during deployment. No need to redo it here.